crates/turborepo-lib/src/turbo_json/parser.rs

```diff
@@ -320,6 +321,100 @@ impl Deserializable for ConfigurationOptions {
     }
 }
 
+impl Deserializable for RawRemoteCacheOptions {
+    fn deserialize(
+        value: &impl DeserializableValue,
+        name: &str,
+        diagnostics: &mut Vec<DeserializationDiagnostic>,
+    ) -> Option<Self> {
+        value.deserialize(RawRemoteCacheOptionsVisitor, name, diagnostics)
+    }
+}
+
+struct RawRemoteCacheOptionsVisitor;
```

Comment:
With this I think we can get rid of the `impl Deserializable for ConfigurationOptions` along with the `ConfigurationOptionsVisitor`

Reply:
I don't think we can. We also deserialize other files via that visitor. The change I made is specific to the `remoteCache` key in `turbo.json`, all other parsing is left unchanged.